lib/dolly/request.rb

```diff
@@ -88,6 +92,10 @@ def values_to_json hash
     def full_path resource
       "/#{database_name}/#{resource}"
     end
+
+    def attachment_path resource, attachment_name
+      "#{full_path(resource)}/#{attachment_name}"
```

Comment:
only thing I see is that it says "The attachment name provided must be a URL encoded string". Is this string encoded?